frontend-legacy/src/components/composite/responsiveDrawer/responsiveDrawer.js

```diff
@@ -29,25 +30,32 @@ const ResponsiveDrawer = ({ handleTabChange }) => {
     setMobileOpen(false);
   };
 
-  const items = [
-    { value: 0, text: "Advisories" },
-    { value: 1, text: "Park Access Status" },
-    { value: 2, text: "Activities & Facilities" },
-    { value: 3, text: "Dates of Operation", href: "/dates/" },
+  const navItems = [
+    { value: 0, text: "Advisories", allowedRoles: ["submitter", "approver"] },
+    {
+      value: 1,
+      text: "Park Access Status",
+      allowedRoles: ["submitter", "approver"],
+    },
+    { value: 2, text: "Activities & Facilities", allowedRoles: ["approver"] },
+    {
```

Comment:
I just forgot why "Park Access Status" is available for both roles, but not "Activities & Facilities". Was it documented somewhere?

Reply:
Park Access Status is view only.  That might have been the reasoning.